jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -6,15 +6,14 @@
  * this file be licensed under the Apache-2.0 license or a
  * compatible open source license.
  */
-
-lib = library(identifier: 'jenkins@8.3.3', retriever: modernSCM([
+lib = library(identifier: 'jenkins@8.4.1', retriever: modernSCM([
```

Comment:
Depends on https://github.com/opensearch-project/opensearch-build-libraries/pull/637

Reply:
Tag is cut and now updated.